blake2/Cargo.toml

```diff
@@ -12,6 +12,7 @@ categories = ["cryptography", "no-std"]
 [dependencies]
 digest = "0.8"
 byte-tools = "0.3"
+byteorder = { version = "1", default-features = false }
```

Comment:
Curious if it might make sense to start requiring Rust 1.32+ and using the built-in endianness conversions rather than the `byteorder` crate

Reply:
I can circle back on this after we bump the `digest` crate and with it MSRV